docs/ai/quickstarts/snippets/text-to-image/azure-openai/Program.cs

```diff
@@ -0,0 +1,83 @@
+// <SnippetConfigClient>
+using Azure;
+using Azure.AI.OpenAI;
+using Microsoft.Extensions.AI;
+
+using Microsoft.Extensions.Configuration;
+using System.Drawing;
+IConfigurationRoot config = new ConfigurationBuilder()
+    .AddUserSecrets<Program>()
+    .Build();
+
+string endpoint = config["AZURE_OPENAI_ENDPOINT"];
+string apiKey = config["AZURE_OPENAI_API_KEY"];
```

Comment:
Technically only the key needs to be in secrets.

Not sure if it's in scope for the example but we could show how to use this stuff with hosting, by using things like `AddAzureOpenAIClient` and `AddImageGenerator` similar to https://github.com/ericstj/imageGeneratorSample/blob/8a976997f5a762885c2fa3e76c206723263c3ded/imageGeneratorSample.Web/Program.cs#L16-L26.

Reply:
Logged https://github.com/dotnet/docs/issues/49445.